README_RISE.md

```diff
@@ -0,0 +1,55 @@
+#####Instrukcja instalacji wtyczki RISE służącej do obsługi interaktywnego pokazu slajdów
+
+
+# UNIX 
+
+1. Pobierz potrzebne pliki z repozytorium https://github.com/damianavila/RISE.git
+```
+cd
+git clone https://github.com/damianavila/RISE.git
+```
+
+2. Do instalacji należy wykorzystać skrypt `setup.py`, jednak wcześniej trzeba ustawić zmienną<br>
+   JUPYTER_CONFIG_DIR dodając do pliku `~/.bashrc` następującą linię:<br>
+   
+   `export JUPYTER_CONFIG_DIR="<home_directory>/.jupyter/:$JUPYTER_CONFIG_DIR"`
+   
+   Przydatna może być też zmienna JUPYTER_PATH:
+   
+   `export JUPYTER_PATH="<home_directory>/.local/share/jupyter/:$JUPYTER_PATH"`
+   
+3. Użyj skryptu `setup.py` aby zainstalować wtyczkę
+
+```
+cd RISE/
+python setup.py install
```

Comment:
Prosze dopisac informacje czy ma to byc python2 czy python3. Moze sie tez zdarzyc ze uzytkownik ma obie wersje.